Description: Fixes the compatibility with the version of plexus-container-default in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/maven-scm-test/src/main/java/org/apache/maven/scm/ScmTestCase.java
+++ b/maven-scm-test/src/main/java/org/apache/maven/scm/ScmTestCase.java
@@ -77,6 +77,11 @@
         scmManager = null;
     }
 
+    protected java.io.InputStream getCustomConfiguration() throws Exception
+    {
+        return null;
+    }
+
     /**
      * @return default location of the test read/write repository
      */
--- a/maven-scm-providers/maven-scm-providers-svn/maven-scm-provider-svnexe/src/test/java/org/apache/maven/scm/provider/svn/svnexe/command/changelog/SvnChangeLogConsumerTest.java
+++ b/maven-scm-providers/maven-scm-providers-svn/maven-scm-provider-svnexe/src/test/java/org/apache/maven/scm/provider/svn/svnexe/command/changelog/SvnChangeLogConsumerTest.java
@@ -56,7 +56,7 @@
     {
         super.setUp();
 
-        logger = getContainer().getLogger();
+        logger = ((org.codehaus.plexus.MutablePlexusContainer) getContainer()).getLogger();
         consumer = new SvnChangeLogConsumer( new DefaultLog(), null );
     }
 
